.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: scala
 scala:
-   - 2.12.6
+   - 2.12.10
```

Comment:
I can't include Scala 2.13 in this list at this point, as Travis CI will then attempt to compile the entire project with 2.13, and then fail because it can't compile the play-2.6 module (where Scala 2.13 is not available)